docker/authzed-smoke.sh

```
@@ -9,6 +9,9 @@ readonly PROJECT_NAME="formbricks-authzed-smoke-${$}"
 readonly AUTHZED_TOKEN="0000000000000000000000000000000000000000000000000000000000000001"
 readonly AUTHZED_DATABASE_PASSWORD="0000000000000000000000000000000000000000000000000000000000000002"
 readonly WRONG_AUTHZED_TOKEN="0000000000000000000000000000000000000000000000000000000000000003"
+readonly SCHEMA_LOG_SENTINEL="Canonical Formbricks authorization schema."
+readonly SMOKE_TEMP_DIR="$(mktemp -d)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)